src/main/Osu.go

```diff
@@ -0,0 +1,107 @@
+package main
+
+import (
+	"github.com/thehowl/go-osuapi"
+	"github.com/bwmarrin/discordgo"
+	"fmt"
+	"strings"
+	"strconv"
+)
+
+func checkBeatmapLink(s *discordgo.Session, m *discordgo.MessageCreate){
+
+	//check if message contains only one beatmap link
+	if strings.Contains(m.Content, "https://osu.ppy.sh/s/") && strings.Contains(m.Content, "https://osu.ppy.sh/b/") || strings.Count(m.Content, "https://osu.ppy.sh/s/") > 1 || strings.Count(m.Content, "https://osu.ppy.sh/b/") > 1{
```

Comment:
`strings.Count(m.Content, "https://osu.ppy.sh/s/") + strings.Count(m.Content, "https://osu.ppy.sh/b/") > 2`

Might be a better way to do this, since else we'll loop over the string 4 times.